lib/db/schema/patch-048-049.sql

```diff
@@ -0,0 +1,187 @@
+CREATE PROCEDURE `createAccount_7`(
+    IN `inUid` BINARY(16) ,
+    IN `inNormalizedEmail` VARCHAR(255),
+    IN `inEmail` VARCHAR(255),
+    IN `inEmailCode` BINARY(16),
+    IN `inEmailVerified` TINYINT(1),
+    IN `inKA` BINARY(32),
+    IN `inWrapWrapKb` BINARY(32),
+    IN `inAuthSalt` BINARY(32),
+    IN `inVerifierVersion` TINYINT UNSIGNED,
+    IN `inVerifyHash` BINARY(32),
+    IN `inVerifierSetAt` BIGINT UNSIGNED,
+    IN `inCreatedAt` BIGINT UNSIGNED,
+    IN `inLocale` VARCHAR(255)
+)
+BEGIN
+    DECLARE EXIT HANDLER FOR SQLEXCEPTION
+    BEGIN
+        ROLLBACK;
+        RESIGNAL;
+    END;
+
+    START TRANSACTION;
+
+    -- Check to see if the normalizedEmail exists in the emails table before creating a new user
+    -- with this email.
```

Comment:
I think originally, I wanted to distinguish the error from emails table and account table. The UI would normally display `Account Exists. Sign In` if email exists on account table. By doing the special error, it just shows `Account Exists`. If secondary emails could sign-in then we could probably use the original error. Or we can perform the check in auth-server via `getSecondaryEmail` and avoid it here.